src/Exception/Exception.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Penny\Exception;
+
+use Exception as BaseException;
+
+class Exception extends BaseException
```

Comment:
I'm not fully sure about this, because https://github.com/pennyphp/penny/pull/97/files#diff-58980ac23a3a9a0b92abd9805eed1e78R37 isn't a runtime exception.

In this PR the `PennyException` is just an interface for all the exceptions defined here, to prevent SPL hierarchy changes and mark them a "thrown by Penny" exceptions.

What do you think @gianarb @wdalmut ?